dags/resources/stages/parse/table_definitions/tornado/TornadoCash_eth_event_Deposit.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Is the `_eth_` part in the table name and file name necessary?

Reply:
You are right. I've removed `eth` in the file name and table name.